compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/JMethodResolver.java

```diff
@@ -897,6 +898,10 @@ private Executable resolveMethod(Class<?> clazz, String name, Class<?>... paramT
     private List<Executable> getExecutables(Class<?> clazz, String methodName, JMethodKind kind) {
 
         if (kind == JMethodKind.CONSTRUCTOR) {
+            if (Modifier.isAbstract(clazz.getModifiers())) {
+                throw new JInteropException(DiagnosticErrorCode.INSTANTIATION_ERROR,
```

Comment:
This error message is the same as the Java error. Shall we verify whether this error message format is in compliance with Ballerina?

Reply:
Modified it as `'Clazz' is abstract, and cannot be instantiated` per discussion.